coding-guide.html

```diff
@@ -19,7 +19,7 @@
 </ul>
 
 <h2>Scala</h2>
-We are following the style guide given <a href="http://davetron5000.github.com/scala-style/ScalaStyleGuide.pdf ">here</a> (though not perfectly). Below are some specifics worth noting:
+We are following the style guide given <a href="http://docs.scala-lang.org/style/ ">here</a> (though not perfectly). Below are some specifics worth noting:
```

Comment:
Yes and no. Yes because it was already there, and I wasn't sure if there was some reason for it. No because I didn't choose for it to be there. I am somewhat removed from HTML-land so I didn't know it this was some convention or something.